package.json

```diff
@@ -64,8 +64,9 @@
     "@hapi/boom": "^9.1.1",
     "@hapi/hapi": "^20.0.1",
     "@hapi/inert": "^6.0.3",
-    "@opencrvs/mosip": "^1.7.0-alpha.13",
+    "@opencrvs/mosip": "^1.7.0-alpha.15",
     "@opencrvs/toolkit": "0.0.32-jr",
+    "@opencrvs/toolkit-legacy": "npm:@opencrvs/toolkit@0.0.30",
```

Comment:
Wow, didn't know you can do this

Reply:
Well I had to do it to make the test compilation pass with the latest version as it was throwing errors with the tennis club membership forms  (not sure why) and add legacy that has `objectHasProperty`